capture.go

```diff
@@ -39,18 +48,27 @@ type Capture struct {
 	// missing and not explicitly set to the empty string.
 	Env []string `json:"env,omitempty"`
```

Comment:
Interesting.  I noticed after making this comment, further down in the code that there is a reference to the parent process's environment.  So, in this code, that's three different ways to which the same thing is referred.  They are all the same thing.  They are all synonyms.  We can decide that's acceptable and move on.  My comment was, to be clear, about consistency in technical writing style, and not a judgement on correctness.  I find, personally, when I am new to a topic and the terminology is lax (or even slightly inconsistent with itself), the documentation is just a little more difficult to understand (comments constituting a form of documentation in this context).  I don't think we're beholden to the upstream author's word choices if a different word choice might make things more clear.  I only intended it as a point to consider.

Reply:
Yeah, I'm a bit torn on this because providing no documentation is unacceptable, but copying leads to confusion. Devin has commented on this exact documentation before because it *is* copied, and some of the context doesn't really line up.